modules/exploits/linux/local/netfilter_priv_esc.rb

```diff
@@ -0,0 +1,138 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GreatRanking
+  include Msf::Post::Common
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::System
+  include Msf::Post::Linux::Kernel
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => 'Netfilter Privilege Escalation',
```

Comment:
Suggest a more verbose name for this module (and module file name).

There are, and will be continue to be, many exploits for Netfilter. We already have a `netfilter_priv_esc_ipv4.rb` local Linux exploit module. That wasn't the first exploitable Netfilter vulnerability. This module won't be the last exploitable Netfilter privilege escalation vulnerability.

Perhaps:

* Netfilter x_tables Heap OOB Write Privilege Escalation